codegen/smithy-dafny-codegen/src/main/java/software/amazon/polymorph/smithygo/localservice/DafnyLocalServiceGenerator.java

```diff
@@ -141,10 +144,20 @@ func NewClient(clientConfig $L) (*$T, error) {
             if (inputShape.hasTrait(UnitTypeTrait.class)) {
                 baseClientCall = "var dafny_response = client.DafnyClient.%s()".formatted(operationShape.getId().getName());
             } else {
+                String dafnyType; 
+                if (inputShape.hasTrait(PositionalTrait.class)) {
+                    Shape inputForPositional = model.expectShape(inputShape.getAllMembers().values().stream().findFirst().get().getTarget());
```

Comment:
Why not `getAllMembers().get(0)` assuming it's a collection?

Reply:
`getAllMembers()` is a map but `getAllMembers().values()` is a collection. However, collection does not have `get` method.